data/state/accountsDB.go

```diff
@@ -26,6 +29,9 @@ type AccountsDB struct {
 	dataTries    TriesHolder
 	entries      []JournalEntry
 	mutOp        sync.RWMutex
+
+	numCheckpoints      uint
```

Comment:
could have used uint32 and for increment function, `atomic.AddUint32()`

Reply:
Done.